### PR TITLE
Add RepositoryCommit and RepositoryBranch for pack

### PR DESCRIPTION
**Pull request by emgarten**
Document commit and branch nuspec additions.

Related: https://github.com/NuGet/Home/issues/6547
Related: https://github.com/NuGet/NuGet.Client/commit/0e4d1b598f350b3dc675018d539114d1328189ef

//cc @rohit21agrawal

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/msbuild-targets.md](https://github.com/emgarten/docs.microsoft.com-nuget/blob/patch-1/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201802162201531975-708%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zk3snNk3QsMbbLFqhht8StN5zFAKI%2BFsl3q3vJ9%2BLfQ%3D&st=2018-02-16T21%3A57%3A42Z&se=2018-03-19T22%3A02%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/msbuild-targets.md [hunk 1]
```diff
@@ -59,8 +59,10 @@ Note that the `Owners` and `Summary` properties from `.nuspec` are not supported
 | IconUrl | PackageIconUrl | empty | |
 | Tags | PackageTags | empty | Tags are semi-colon delimited. |
 | ReleaseNotes | PackageReleaseNotes | empty | |
-| RepositoryUrl | RepositoryUrl | empty | |
-| RepositoryType | RepositoryType | empty | |
+| Repository/Url | RepositoryUrl | empty | Repository URL used to clone or retrieve source code. Example: *https://github.com/NuGet/NuGet.Client.git* |
+| Repository/Type | RepositoryType | empty | Repository type. Examples: *git*, *tfs*. |
+| Repository/Branch | RepositoryBranch | empty | Optional repository branch information. Example: *master* (NuGet 4.7.0+) |
```
Comment (rohit21agrawal) [inline docs/reference/msbuild-targets.md]: can we add that this will only be picked up if url is provided?

Comment (emgarten) [reply docs/reference/msbuild-targets.md]: Added

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/msbuild-targets.md](https://github.com/emgarten/docs.microsoft.com-nuget/blob/patch-1/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C2%5C16%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201802162218128022-708%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9T7syexKiDI8gY0QoJV81I3sALBJ%2BkkWtMTA4cqLWPw%3D&st=2018-02-16T22%3A14%3A06Z&se=2018-03-19T22%3A19%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**